README.md

```diff
@@ -32,6 +32,37 @@ fastify.listen(3000, err => {
 })
 ```
 
+### Content-Security-Policy Nonce
+
```

Comment:
Let's document some of this is implemented in this module instead to provide the decorated values as decorators in the reply.

Reply:
Would you mind to explain more? Should I elaborate more on how this options works?

Reply:
This package is called `-helmet` as it wraps helmet itself. This feature is partially implemented inside this module, we should list the differences.

Reply:
Update.
I also add an example about how to use helmet option only.